collector/benchmarks/token-stream-stress/perf-config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "cargo_opts": "--bin token-stream-stress",
+    "runs": 1
```

Comment:
Is there a reason for running this just once? Speed? Generally I think we should try to keep the default of 3 runs and only adjust lower if it's a super long benchmark (which this doesn't seem to be).

Reply:
No good reason. I think I unwittingly copied that `perf-config.json` file from one of the benchmarks that only runs once. I will change it on Monday.